src/adapter/src/catalog/migrate.rs

```diff
@@ -142,3 +146,28 @@ fn csr_url_path_rewrite(stmt: &mut mz_sql::ast::Statement<Raw>) {
 // ****************************************************************************
 // Semantic migrations -- Weird migrations that require access to the catalog
 // ****************************************************************************
+
+/// Add normalization to all CREATE SECRET statements.
```

Comment:
Can you add a doc comment for if/when this migration can be removed?

Reply:
Done